### DISCUSSION TITLE
SX1268 Support

### DISCUSSION THREAD

**Discussion by au51gtm** [dl9rdz/rdz_ttgo_sonde#347] [Q&A]
I recently purchased a 433mHz T-Beam with the CH9102 / SX1268 / NEO-M8N chipset combination.  After loading the software, I noticed that the fingerprint was not identified.  The software largely worked normally, including the GPS, however the receiver did not function.  Forgive me if this question has been asked elsewhere, but are you intending to incorporate support for these new T-Beam release chipsets into rdz_ttgo?

**Comment by LY3OO** [comment]
https://github.com/dl9rdz/rdz_ttgo_sonde/issues/338#issue-1490505115

**Reply by au51gtm** [reply]
Thank you, LY3OO, for your quick reply and for the link to the bug report.  Very much appreciated.
